### PR TITLE
Adding note about python server deprecation

### PR DESCRIPTION
**Pull request by iloveitaly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,19 +5,17 @@ aw-server
...
 ## Build and Install
 
-To install program and dependencies, Poetry is required.
+To install program and dependencies, Poetry is required. You'll want to activate a `venv` before building.
 
 ```bash
 make build
 ```
 
-The `aw-server` binary should now be available to you in your PATH (if it is set correctly).
```
Comment (ErikBjare) [inline README.md:19]: Why remove this?

Comment (iloveitaly) [reply README.md:19]: I'm not sure, I'll add it back in.

**Comment by iloveitaly** [comment]
@ErikBjare want to take another look?

**Comment by ErikBjare** [comment]
@iloveitaly Sorry for being slow, my notifications have been a mess.

**Pull request merged**